scripts/clang_format_wrapper.sh

```diff
@@ -67,7 +67,7 @@ echo "formatting"
 
 cd ${TEMP_DIR}
 
-clang-format ${CLANG_FORMAT_FLAGS} ${FILE_NAMES}
+clang-format-8 ${CLANG_FORMAT_FLAGS} ${FILE_NAMES}
```

Comment:
still there.

Reply:
I have to edit the file each time since the CMake does not work correctly, and it is added each time I do `git commit -a` and forget to stash the file.